project/npda/general_functions/session.py

```diff
@@ -15,14 +15,39 @@
 logger = logging.getLogger(__name__)
 
 
+def get_submission_actions(pz_code, audit_year):
+    Submission = apps.get_model("npda", "Submission")
+
+    submission = Submission.objects.filter(
```

Comment:
See the TODO in `views/patient.py` - this pattern is repeated enough I think it's maybe worth being a custom manager method?

Reply:
More involved than I thought so pushing it out to https://github.com/rcpch/national-paediatric-diabetes-audit/issues/533